soccer/gameplay/tactics/penalty.py

```diff
@@ -47,18 +48,25 @@ def execute_setup(self):
         penalty_mark = robocup.Point(
             0, constants.Field.Length - constants.Field.PenaltyDist)
         backoff = 0.5
-        if main.ball().pos.near_point(penalty_mark, 0.5):
-            self.robot.move_to(main.ball().pos + (main.ball(
-            ).pos - robocup.Point(0, constants.Field.Length).normalized()) *
-                               backoff)
-        else:
-            self.robot.move_to(penalty_mark - robocup.Point(0, backoff))
+
+        self.robot.disable_avoid_ball()
+
+        self.robot.move_to(main.ball().pos - robocup.Point(0, backoff))
```

Comment:
I'm debating whether to keep this or the old behavior, thoughts?

The difference is that the old behavior sticks to the kickoff point on the goal arc, while the new one tracks soley the ball. There might be rule implications I don't know that would affect this decision though.